src/main/java/org/thoughtcrime/securesms/util/IntentUtils.java

```diff
@@ -21,7 +25,11 @@ public static boolean isResolvable(@NonNull Context context, @NonNull Intent int
 
   public static void showBrowserIntent(Context context, String url) {
```

Comment:
What about renaming it to `showInBrowser()`? `showBrowserIntent()` sounds to me like it will return an intent that shows the URL in the browser.

Reply:
Good idea!